### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by inducer**
cc @matthiasdiener

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,27 +21,6 @@ OpenCL compilation, etc).
 
 * `Documentation <https://documen.tician.de/pytato>`__ (read how things work)
 
-Example::
```
Comment (matthiasdiener) [inline README.rst:24]: Imho, it might be worthwhile to add at least a link to an example.

Comment (inducer) [reply README.rst:24]: 7a7da8a?

Comment (matthiasdiener) [reply README.rst:24]: looks good!

**Review by kaushikcfd — APPROVED** [review]
LGTM.

**Pull request merged**